02_bubble/nickolee/bubblesort.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+// general sorting algorithm that sorts by "bubbling" smaller numbers to the left
+func bubbleSort(s []int) []int {
+	result := make([]int, len(s))
+	copy(result, s)
+
+	unsorted := true
+	for unsorted {
+		// perform the check to see if sorting is complete to exit for loop
```

Comment:
Comments like this usually go on the same line or before the check.

Reply:
Fixed this